docs/commands/preview-docs.md

```diff
@@ -1,39 +1,98 @@
+
+## Examples
+
+### Entrypoints
+
+The command behaves differently depending on how you pass an entrypoint to it and whether the [configuration file](#custom-configuration-file) exists. There are the following options:
 
-```shell
-Positionals:
-  entrypoint                                                 [string] [required]
+#### Pass entrypoint directly
 
-Options:
-  --version                Show version number.                        [boolean]
-  --help                   Show help.                                  [boolean]
-  --port, -p               Preview port.                [number] [default: 8080]
-  --skip-preprocessor      Ignore certain preprocessors.                 [array]
-  --skip-decorator         Ignore certain decorators.                    [array]
-  --use-community-edition  Force using Redoc CE for docs preview.      [boolean]
-  --force, -f              Produce bundle output even when errors occur.
-                                                                       [boolean]
-  --config                 Specify path to the config file.             [string]
+```bash
+openapi preview-docs openapi/openapi.yaml
 ```
 
+In this case, `preview-docs` will preview the definition that was passed to the command. The configuration file is ignored.
 
-### How to preview the docs on a custom port
+#### Pass entrypoint without extension
 
-By default, without providing a port, the preview starts on port 8080, and can be accessed at `http://localhost:8080`.
+You can omit entrypoint's file extension when executing the `preview-docs` command. In this way, you can reference either `.yaml` or `.json` file.
 
-This command starts a preview on port 8888, and you can access the docs at `http://localhost:8888` after running it.
+```bash
+# preview-docs will preview either petstore.yaml or petstore.json file in the current working directory
+openapi preview-docs petstore
+# preview-docs will preview either sandbox.yaml or sandbox.json file in the openapi/extra directory
+openapi preview-docs openapi/extra/sandbox
+```
+
+#### Pass entrypoint via configuration file
+
+Instead of a full path, you can use an alias assigned in your `apiDefinitions` within your `.redocly.yaml` configuration file as entrypoints. For example, `petstore`:
 
+```bash command
+openapi preview-docs petstore
+```
+
+```yaml .redocly.yaml
+apiDefinitions:
+  petstore: ./openapi/petstore-definition.json
+```
 
-```shell
+In this case, after resolving the path behind the `petstore` alias (see the `.redocly.yaml` tab), `preview-docs` will preview the `petstore.json` definition file. The presence of the `.redocly.yaml` configuration file is mandatory.
+
+### Custom configuration file
+
+By default, the CLI tool looks for a `.redocly.yaml` configuration file in the current working directory. Use the optional `--config` argument to provide an alternative path to a configuration file.
+
+```bash
+openapi preview-docs --config=./another/directory/config.yaml
```

Comment:
question: Will this work even if the config file has a custom name (if it's not called `.redocly.yaml`)?

Reply:
Yes, it will.
I tested that case and there is the relevant section about that in the [lint command PR](#360). I will transfer it from there

Reply:
Awesome, thanks